applications/blockchainia.md

```diff
@@ -0,0 +1,139 @@
+# Blockchainia
+
+- **Team Name:** Blockchainia
+- **Payment Address:** 0xf246aede3d892234b52c9bb6f246ab0ac8c0491d (DAI)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview
+
+### Overview
+
+The Blockchainia project is a decentralized game distribution and online multiplayer infrastructure. 
+
+ As players (our eventual community) progress through seasonally released levels, their in game actions will be logged to our substrate built parachain (or parathread, we are still doing R&D to see what works best for economy and the end user experience). Our parachain will consist of open-source palettes that enable game servers to interact with parachains in real time. We are creating a web3 enhanced online multiplayer game alongside this project to showcase the features a decentralized game-iverse allows. Our expanding pallets will be tested by the "seasonal" deliverables of our online multiplayer video game. In a new value proposition for consumers, the game will be downloadable for free. Players will be required to purchase an NFT to server as their in-game avatar. Logging in with a wallet containing an NFT will allow you to connect to an online infrastructure to play  web3 enhanced games with a nominal pay-to-play and pay-to-continue fee. 
+
+This project will involve creating an api for interactions between our off-chain game server and parachain. Custom pallets will relay real-time in-game events between the blockchain and off-chain worker. Off-chain servers must be authorized through a DAO that runs our parachain, which will ensure the integrity of those who host online competative gameplay. Community members who run servers will earn a portion of each players pay-to-play fee, a portion of which will pay the parachain overhead incurred by the server. 
+
+This project will allow us to test our parachain and online multiplayer infrastructure to prepare for market research.
+
+With a successful initial MVP, we can perform market research in the nearby Business Research Center at the University of Pittsburgh. This research will allow us to create pallettes that encourage consumer engagement and interaction. With the data output from these studies and focus groups we can publish estimated potential earnings to entice other developers to use our infrastructure and distribution model. 
```

Comment:
What do you mean by _pallettes_?

Reply:
Sorry, that was a typo. We plan to perform market research through a consumer research lab available at the University of Pittsburgh's Business Research Center. Along with the feedback from our community through Discord, we will run behavioral studies through the research center to verify our hypotheses on how to better engage gamers with functionality they find entertaining or might otherwise increase interest, engagement, and evangelism.